test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,25 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
+        // objects with at least one null
+        assertAnyNull(null, "a", "b", null);
+        assertAnyNull(1, Integer.valueOf(1), null);
+        //assertAnyNull(null);
```

Comment:
You can leave this in the test because it is testing a list with a single entry. No need to comment it out.

Reply:
Oh! Java will read the null value as a null instead of an array with a single value of null. As a result, the isAnyNull function will have an exception.(Referring to the varargs in the java function). I did not know how to get around it and I forgot to ask during the tutorial. 

![image](https://user-images.githubusercontent.com/28092058/45255481-b4e49100-b3b9-11e8-86e4-f85d2cb9ba79.png)

Reply:
There are two ways:
1) Typecast null as an Object: `assertAnyNull((Object) null)`
2) Jeffrey's method: insert a conditional statement in `isAnyNull()` method to check if `items == null`. If it is return true, else false.

Reply:
Thank you!